src/fdns/server.c

```diff
@@ -419,6 +419,12 @@ static int load_file(const char *fname) {
 
 	int linecnt = 0; // line counter
 	DnsServer **ptr = &slist;
+
+  // go to the end of the list
+  while (*ptr != NULL) {
+    ptr = &(*ptr)->next;
+  }
+
```

Comment:
This file uses tabs for indention and not 2-spaces.

Reply:
Sorry, I forgot my IDE reformatted the code. It is now fixed :-)